homeassistant/components/netgear/router.py

```diff
@@ -186,6 +193,13 @@ async def async_update_device_trackers(self, now=None) -> None:
 
         return new_device
 
+    async def async_get_traffic_meter(self) -> None:
+        """Get the traffic meter data of the router."""
+        if self.traffic_meter_entities > 0:
```

Comment:
Instead of guarding this call here, make a separate coordinator for the traffic meter data. Then the coordinator will only fetch data as long as there are listeners (enabled entities).

Reply:
How will we prevent the two coordinators from making a call to the router at the same time?
I do not want to overload the SOAP server in the router.

Reply:
I guess I need to add PARALLEL_UPDATES = 1, but in which file and where do I need to add this to take effect for the two seperate coordinators?
https://developers.home-assistant.io/docs/integration_fetching_data/#request-parallelism

Reply:
Add an `asyncio.Lock` on the router that gurmard the api calls.

Reply:
done